iOSClient/NCAccount.swift

```diff
@@ -58,19 +60,24 @@ class NCAccount: NSObject {
         NextcloudKit.shared.getUserProfile(account: account) { account, userProfile, _, error in
             if error == .success, let userProfile {
                 /// Login log debug
-                NextcloudKit.shared.nkCommonInstance.writeLog("[DEBUG] Create new account \(account) with user \(user) and userId \(userProfile.userId)")
+                NextcloudKit.shared.nkCommonInstance.writeLog(info: "Got user profile, creating new account \(account) with user \(user) and userId \(userProfile.userId)")
```

Comment:
Here we don't use "did get user profile", but a more "human" response. I prefer this, and it's mostly how we log throughout the app.